google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java

```diff
@@ -114,6 +115,14 @@ public static WriteOption labels(Map<String, String> labels) {
     public static WriteOption destination(LogDestinationName destination) {
       return new WriteOption(OptionType.LOG_DESTINATION, destination);
     }
+
+    /**
+     * Returns an option to opt-out automatic population of log entries metadata fields that are not
+     * set.
+     */
+    public static WriteOption autoPopulateMetadata(boolean autoPopulateMetadata) {
+      return new WriteOption(OptionType.AUTO_POPULATE_METADATA, autoPopulateMetadata);
```

Comment:
I dont see in this PR where it is used - what is a plan?

Reply:
Mind that it is merged into `minherz/structured_logging` branch and not into `main`. The auto population implementation will be added and then will use the configuration control its execution.